src/utils/findRanges.ts

```diff
@@ -1,30 +1,32 @@
-/**
+/*!
  * SPDX-FileCopyrightText: 2020 Nextcloud GmbH and Nextcloud contributors
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
 
+export interface Range {
+	start: number
+	end: number
+}
```

Comment:
It is used nowhere, and it intersects with and existing DOM interface `Range`.
Do we need it?